### DISCUSSION TITLE
On a Le Potato card?

### DISCUSSION THREAD

**Discussion by blogdemoi** [Nachtzuster/BirdNET-Pi#168] [Q&A]
Hi,
I have been running the original Birdnet-Pi on a Le Potato card for a long time and now I wish to migrate to your fork. Will it work on a Le Potato Card?
I am running Bullseye.
Thank you.

**Comment by blogdemoi** [comment]
So I updated my previous install, rebooted my Le Potato, checked that it worked fine and launched the migration process.

```
git remote remove origin
git remote add origin https://github.com/Nachtzuster/BirdNET-Pi.git
```

Then:
`./scripts/update_birdnet.sh`

Now I am stuck there, nothing is happening in my cli:
![screenshot_2024-08-08-13:24:35](https://github.com/user-attachments/assets/526f6fc6-2dc3-4e37-88cb-c78db5001bd0)

**Comment by blogdemoi** [comment]
I rebooted, it seems to be working but I am not sure the migration was successful, how to be sure?

It seems to have migrated as I have new icons (links towards allaboutbirds, wikipedia, etc.) which I didn't have with the original BirdnetPI install, however my "Daily charts" don't include all species, just 10 as before...
![FireShot Capture 004 - Garches France - 10 0 242 72](https://github.com/user-attachments/assets/42a0adbb-0f61-4224-9097-75ab0011eaa2)

![FireShot Capture 003 - Garches France - 10 0 242 72](https://github.com/user-attachments/assets/2b867d07-8c09-49a3-aa2d-892ddad15720)

Thank you for any help.

**Comment by blogdemoi** [comment]
I re-ran .`/scripts/update_birdnet.sh` and this time it seems it was completed successfully as my "Daily charts" include now all species.

However in the "Service" tab, even though all services are active, I have a warning:

> BirdNET Analysis (stalled - backlog of 353 files in ~/BirdSongs/StreamData/)

![screenshot_2024-08-08-18:27:20](https://github.com/user-attachments/assets/d0b51480-1d3c-446f-be10-17322557bc9f)

Do I have to simply wait? Or do something else?

Also could you explain what is?:

> Ram drive (!experimental!) (inactive)

Thank you.

**Reply by Emmo213** [reply]
> Also could you explain what is?:
>
>>Ram drive (!experimental!) (inactive)

See discussion #25

**Reply by blogdemoi** [reply]
> > Also could you explain what is?:
> > > Ram drive (!experimental!) (inactive)
> 
> See discussion #25

Thank you, so I prefer not to activate this for now :-) as I am not using a Pi but a Le Potato card.

As for my "backlog of 353 files" is seems to be solved by now. So I may consider my migration a success! Thank you @Nachtzuster , and everybody, for this fork.

My Birdnet-Pi is public (in France) but I am on a slow Internet connection: https://birdnetpi.arnauld.org/

**Reply by Emmo213** [reply]
Did you have to do anything to process the files or just give it time?

**Reply by blogdemoi** [reply]
Just give it time.
